src/modules/FavoritesMovies/FavoritesMovies.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react"
+
+import { useAppDispatch, useAppSelector } from "src/hooks/hooks"
+import { Movie } from "src/modules/common/Movies.tsx/Movie"
+import { IMovie } from "src/modules/movies/interface/imovie.interface"
+import {
+  favoritesMoviesListSelector,
+  loadingDataFromTheServerFavoritesMovie,
+} from "src/modules/movies/store/favoritesMovies/favoritesMoviesSlice"
+
+import styles from "./favoritesMovies.module.scss"
+
+const FavoritesMovies = () => {
+  const dispatch = useAppDispatch()
+  // const [favoritesMoviesId, setFavoritesMoviesId] = useState([])
+  const favoritesMoviesList = useAppSelector(favoritesMoviesListSelector)
+  const [stateHeart, setStateHeart] = useState(true)
+  const getMoviesIdFromLocalStorage = localStorage.getItem("favoritesMoviesIdList")
+
+  useEffect(() => {
+    if (getMoviesIdFromLocalStorage) {
```

Comment:
```suggestion

  useEffect(() => {
    const getMoviesIdFromLocalStorage = localStorage.getItem("favoritesMoviesIdList")
    if (getMoviesIdFromLocalStorage) {
```

Ты должен сделать это в useEffect. А не при маунте компонента. Ключ `favoritesMoviesIdList` вынеси в константу